source/direct_debit/index.html.md.erb

```diff
@@ -5,248 +5,126 @@ weight: 80
-    <th>Description</th>
-    </tr>
-    <tr>
-    <td><code>SUCCESS</code></td>
-    <td>Mandate successfully set up.</td>
-    </tr>
-    <td><code>FAILED</code></td>
-    <td>User attempted to make a payment but the mandate was not set up successfully.</td>
-    </tr>
-    <td><code>CANCELLED</code></td>
-    <td>The user has cancelled their mandate.</td>
-    </tr>
-    <td><code>CANCELLED - USER NOT ELIGIBLE</code></td>
-    <td>The user indicated that they are not the sole signatory on the bank account, so the mandate setup could not be completed.</td>
-    </tr>
-    <td><code>ERROR</code></td>
-    <td>Something went wrong with GOV.UK Pay or the underlying Payment Service Provider.</td>
-    </tr>
-  </table>
-</div>
-
-#### Possible agreement state transitions
-
-An agreement changes states as it moves through its lifecycle.
+1. Select your account in the __My services__ page.
 
-<br>
+1. Select **Settings**.
 
-<div style = "overflow-x:auto;">
-  <table style="width:100%">
-    <tr>
-    <th nowrap>Original state</th>
-    <th nowrap>New state</th>
-    <th>Meaning</th>
-    </tr>
-    <tr>
-    <td nowrap><code> CREATED </code></td>
-    <td nowrap><code> STARTED </code></td>
-    <td> Payer has visited the `next_url` and landed on the GOV.UK Pay page to enter their account details. </td>
-    </tr>
-    <tr>
-    <td nowrap><code> STARTED </code></td>
-    <td nowrap><code> PENDING </code></td>
-    <td> Payer has provided their account details, and confirms their consent to the agreement. <code>PENDING</code> is only used with delayed payment types such as Direct Debit. This indicates that payer consent has been given but the agreement has not yet been confirmed with the issuing bank. </td>
-    </tr>
-    <tr>
-    <td nowrap><code> STARTED </code></td>
-    <td nowrap><code> CANCELLED - USER NOT ELIGIBLE </code></td>
-    <td> Payer has indicated that they are not the sole signatory for their bank account, and has selected the link to go back and try a different payment method. </td>
-    </tr>
-    <tr>
-    <td nowrap><code> PENDING </code></td>
-    <td nowrap><code> ACTIVE </code></td>
-    <td> The issuing bank has confirmed that funding source (for example, Direct Debit mandate) is valid. </td>
-    </tr>
-    <tr>
-    <td nowrap><code> PENDING </code></td>
-    <td nowrap><code> FAILED </code></td>
-    <td> The issuing bank has rejected the funding source. Possible reasons: incorrect bank account details, bank account requires multiple signatories to set up a Direct Debit. </td>
-    </tr>
-    <tr>
-    <td nowrap><code> PENDING </code></td>
-    <td nowrap><code> CANCELLED </code></td>
-    <td> Cancelled by the payer. </td>
-    </tr>
-    <tr>
-    <td nowrap><code> ACTIVE </code></td>
-    <td nowrap><code> CANCELLED </code></td>
-    <td> Cancelled by the payer. </td>
-    </tr>
-  </table>
-</div>
+1. Select **Link GoCardless Merchant Account** and follow the on-screen instructions.
```

Comment:
It can also be done with a test gocardless account if they want to test against a sandbox gocardless account. There are 3 different types of account they could have "test", "Gocardless test" and "Gocardless live".

The instructions are out of date though as this changed recently, now they select the account and then on the dashboard for that account they are presented with instructions for linking the account and a big green button to press.

![Screen Shot 2019-06-25 at 10 30 33](https://user-images.githubusercontent.com/5648592/60087127-4b5d2580-9734-11e9-9aa4-7751c76ac9e9.png)

Reply:
I've reverted to the original version of the instructions as I think they're still correct, based on @stephencdaly's comment.